.github/workflows/build.yml

```
@@ -28,23 +29,31 @@ jobs:
       with:
         repository: ruby/ruby
         path: ruby
+        fetch-depth: 0
     - name: Set latest_commit
       id: latest_commit
       working-directory: ruby
       run: echo "commit=$(git rev-parse HEAD)" >> $GITHUB_OUTPUT
-
+    - name: Set latest commit (3.4-asan)
+      id: latest_commit_3_4_asan
+      working-directory: ruby
+      run: |
+        git checkout ruby_3_4
+        echo "commit=$(git rev-parse HEAD)" >> $GITHUB_OUTPUT
     - name: Check if latest commit already built
       uses: actions/github-script@v7
       id: check_commit
       with:
         script: |
           const latestDevCommit = "${{ steps.latest_commit.outputs.commit }}"
+          const latest34Asan = "${{ steps.latest_commit_3_4_asan.outputs.commit }}"
```

```suggestion
          const latest34ASan = "${{ steps.latest_commit_3_4_asan.outputs.commit }}"
```
maybe since the official spelling is `ASan`

---

Fixed in e5b18c936f1590c6da7fda9a8795e621ba8d049c